test/functional/wallet_sendall.py

```diff
@@ -379,6 +379,22 @@ def sendall_with_maxconf(self):
         assert_equal(len(self.wallet.listunspent()), 1)
         assert_equal(self.wallet.listunspent()[0]['confirmations'], 6)
 
+    @cleanup
+    def sendall_spends_unconfirmed_change(self):
+        self.log.info("Test that sendall spends unconfirmed change")
+        self.add_utxos([17])
+        self.wallet.sendtoaddress(self.remainder_target, 10)
+        assert_greater_than(self.wallet.getbalances()["mine"]["trusted"], 6)
+        self.test_sendall_success(sendall_args = [self.remainder_target])
```

Comment:
In "rpc, test: test sendall spends unconfirmed change and unconfirmed inputs when specified" 69408120a64316624b9ec379771dbac142dc6646, perhaps you could confirm here that the wallet is now empty:

```
        assert_equal(self.wallet.getbalance(), 0)
```

Reply:
Done